Automattic/wp-calypso
# Domains: Mapping and Transfer pages standardization, take two

* Change pricing display on Transfers page to match placement from Mapping page
* Disable "Add" button on Mapping page until a proper domain name is input, a la Transfers page
* Moves Transfer fine print/Learn More underneath the form to put the focus on the form/pricing.

Is it OK to move that explanatory text from the Transfer page underneath? Does the pricing stand out enough? What do y'all think?

**Before this PR**

<img width="745" alt="screen shot 2018-07-26 at 11 32 50 am" src="https://user-images.githubusercontent.com/2124984/43272346-c34b20b6-90c7-11e8-9741-4e7bb2f6a699.png">

<img width="751" alt="screen shot 2018-07-26 at 11 33 26 am" src="https://user-images.githubusercontent.com/2124984/43272348-c35c9256-90c7-11e8-93d7-fac33a2fcfa7.png">

**After this PR**

Lots of visual changes here, so I've added everything to a gallery: https://cloudup.com/c0giKp1KWjo

**Steps to test**

* Switch to this PR and navigate to `/start/domains/use-your-domain/`
* Click "Transfer to WordPress.com" to note design changes; make sure the form still works.
* Click "Buy Domain Mapping" to note design changes; make sure the form still works.
